src/ops/inventory/ec2inventory.py

```diff
@@ -188,18 +200,19 @@ def get_host_info_dict_from_instance(self, instance):
             key = self.to_safe('ec2_' + key)
 
             # Handle complex types
-            # state/previous_state changed to properties in boto in https://github.com/boto/boto/commit/a23c379837f698212252720d2af8dec0325c9518
+            # state/previous_state changed to properties in boto in
+            # https://github.com/boto/boto/commit/a23c379837f698212252720d2af8dec0325c9518
             if key == 'ec2__state':
                 instance_vars['ec2_state'] = instance.state or ''
                 instance_vars['ec2_state_code'] = instance.state_code
             elif key == 'ec2__previous_state':
                 instance_vars['ec2_previous_state'] = instance.previous_state or ''
                 instance_vars['ec2_previous_state_code'] = instance.previous_state_code
-            elif type(value) in integer_types or type(value) == bool:
+            elif type(value) in integer_types or isinstance(value, bool):
                 instance_vars[key] = value
             elif type(value) in string_types:
                 instance_vars[key] = value.strip()
-            elif type(value) == type(None):
+            elif isinstance(value, type(None)):
```

Comment:
would `elif value is None:` be more clear here? As suggested in the example from https://github.com/adobe/ops-cli/issues/73

Reply:
I changed it, but in my opinion mixing ```isinstance()``` with ```is None``` is not very clear. Unfortunately, the ```is <type>``` statement only works with ```None``` type.